packages/core/src/components/forms/numericInput.tsx

```diff
@@ -236,8 +246,11 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
             const decrementButton = this.renderButton(
                 NumericInput.DECREMENT_KEY, NumericInput.DECREMENT_ICON_NAME, this.handleDecrementButtonClick);
 
+            const buttonGroupClasses = classNames(Classes.BUTTON_GROUP, Classes.VERTICAL, Classes.FIXED, {
+                [Classes.LARGE]: large,
```

Comment:
`pt-button-group` does support the `pt-large` modifier. Here's the code from http://blueprintjs.com/docs/#core/components/button-group.css-api:

```
<div class="pt-button-group pt-large">
  <a class="pt-button pt-icon-database" tabindex="0" role="button">Queries</a>
  <a class="pt-button pt-icon-function" tabindex="0" role="button">Functions</a>
  <a class="pt-button" tabindex="0" role="button">
    Options <span class="pt-icon-standard pt-icon-caret-down pt-align-right"></span>
  </a>
</div>
```